test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void iaAnyNull() throws Exception {
+        // empty list
+        assertContainsNoNull();
+
+        // all objects are not null
+        assertContainsNoNull("abc", "ab", "a");
+        assertContainsNoNull(1, 2);
+
+        // only one null
+        assertContainsNull(null, 1);
+        assertContainsNull("", null);
+        assertContainsNull("", null, "abc");
+
+        // some are null
+        assertContainsNull("abc", null, "abc", null, null);
+        assertContainsNull("abc", "", "ABC", null);
+        assertContainsNull(null, "", "abc", null, "a", "abc");
```

Comment:
Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.